libs/features/japanese-word-wrap.js

```diff
@@ -134,6 +134,7 @@ export async function applyJapaneseLineBreaks(config, options = {}) {
       budouxExcludeElements.has(el)
       || isWordWrapApplied(el)
       || (isFirefox() && hasFlexOrGrid(el))
+      || (el.lang && el?.lang !== 'ja')
```

Comment:
```suggestion
      || (el?.lang && el?.lang !== 'ja')
```
or we can remove ```?``` from both

Reply:
Committed the suggestion.